src/test/java/resources/batch.feature

```diff
@@ -32,4 +32,14 @@ Feature: Batch Details API test
     When I call the auth endpoint with username: "mifos" and password: "password"
     Then I should get a valid token
     When I call the batch details API with expected status of 200
-    Then I should get non empty response
\ No newline at end of file
+    Then I should get non empty response
+
+  Scenario: Batch summary with failure percent API Test
+    Given I have a batch id from previous scenario
+    And I have tenant as "gorilla"
+    When I call the batch transactions endpoint with expected status of 200
+    Then I should get non empty response
+    When I call the auth endpoint with username: "mifos" and password: "password"
```

Comment:
why do we need this step?
we do not have the authentication any more

Reply:
This step is optional. Based on an auth enabled flag, this step will be skipped